scripts/BackupRetentionScript/README.md

```diff
@@ -0,0 +1,75 @@
+
+# About the scripts:
+ Service Fabric Backup Restore Service (https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-backuprestoreservice-quickstart-azurecluster),
+  currently in preview, allows you to take periodic backups of your stateful service. Depending the on the frequency of your backup interval, backups can really grow fast. 
+ While we work on providing retention support integrated with the service, this script would help you manage your storage till that time. It allows you to delete backups older than a specified time.
+ 
+## Usage
+This script support both the storage types supported by Backup Restore service:
+1) Azure blob store
+2) File Share
+
+ ## How to use with Azure Storage:
+ 
+ You can run the script by providing single azure storage connection string as parameter or you can also provide azure account name and key separately.
+ Examples:
+1) The below example will delete all the backups in the storage(specifed with connection string) of the service with serviceid "WebReferenceApplication~RestockRequestManager".
+```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -ConnectionString "your-connection-string" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -ClientCertificateThumbprint "Client#Certificate#Thumbpring" -ServiceId "WebReferenceApplication~RestockRequestManager"
+```
+2) In this example, passing storage account name and key separately. 
+```powershell
+ .\RetentionScript.ps1 -DateTimeBefore "2018-06-18 23.44.03Z" -StorageAccountName "storageaccountname" -StorageAccountName "storgeAccountName" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -ClientCertificateThumbprint "Client#Certificate#Thumbpring" -ServiceId "WebReferenceApplication~RestockRequestManager"
+ ```
+ 3) You can also filter the container for cleanup. This example will delete backup data for all the partitions available on the container with container name "brstorage":
+```powershell
+ .\RetentionScript.ps1 -ContainerName "brstorage"-DateTimeBefore "2018-06-18 23.44.03Z" -StorageAccountName "storageaccountname" -StorageAccountName "storgeAccountName" -ClusterEndPoint "clustername.centralus.cloupapp.azure.com:19080" -ClientCertificateThumbprint "Client#Certificate#Thumbpring"
+ ```
+ 3) Similarly you can also filter the application by providing the ApplicationId. If Application name is "fabric:/WebReferenceApplication" then,
+ 	  application id is "WebReferenceApplication":
```

Comment:
>"WebReferenceApplication" [](start = 22, length = 25)

Instead of using double quotes, you should use markup syntax to achieve required highlighting of word/phrase.